pkg/asset/agent/image/agentimage.go

```diff
@@ -50,6 +60,14 @@ func (a *AgentImage) Generate(dependencies asset.Parents) error {
 	}
 	a.volumeID = volumeID
 
+	if strings.ToLower(string(agentManifests.AgentClusterInstall.Spec.PlatformType)) == external.Name {
```

Comment:
If this is just temporary until the platform type is defined in assisted-service then it would be good to have a comment indicating that.
Personally I'd prefer to see a hard-coded value rather than depend on transformations that may not hold in other cases.